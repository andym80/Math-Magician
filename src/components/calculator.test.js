import React from 'react';
import renderer from 'react-test-renderer';
import Calculator from './calculator';

it('renders to match the test', () => {
    const tree = renderer.create(<Calculator />).toJSON();
    expect(tree).toMatchSnapshot();
  });

