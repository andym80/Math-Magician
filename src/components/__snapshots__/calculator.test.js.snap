// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders to match the test 1`] = `
<div
  className="container"
>
  <span
    className="spantitle"
  >
     Let's do some math!
  </span>
  <form>
    <div
      className="display"
    >
      0
    </div>
  </form>
  <div
    className="keypad"
  >
    <button
      className="firstline"
      onClick={[Function]}
      type="button"
    >
      AC
    </button>
    <button
      className="firstline"
      onClick={[Function]}
      type="button"
    >
      +/-
    </button>
    <button
      className="firstline"
      onClick={[Function]}
      type="button"
    >
      %
    </button>
    <button
      className="extra"
      onClick={[Function]}
      type="button"
    >
      รท
    </button>
    <button
      className="secondline"
      onClick={[Function]}
      type="button"
    >
      7
    </button>
    <button
      className="secondline"
      onClick={[Function]}
      type="button"
    >
      8
    </button>
    <button
      className="secondline"
      onClick={[Function]}
      type="button"
    >
      9
    </button>
    <button
      className="extra"
      onClick={[Function]}
      type="button"
    >
      x
    </button>
    <button
      className="thirdline"
      onClick={[Function]}
      type="button"
    >
      4
    </button>
    <button
      className="thirdline"
      onClick={[Function]}
      type="button"
    >
      5
    </button>
    <button
      className="thirdline"
      onClick={[Function]}
      type="button"
    >
      6
    </button>
    <button
      className="extra"
      onClick={[Function]}
      type="button"
    >
      -
    </button>
    <button
      className="fourthline"
      onClick={[Function]}
      type="button"
    >
      1
    </button>
    <button
      className="fourthline"
      onClick={[Function]}
      type="button"
    >
      2
    </button>
    <button
      className="fourthline"
      onClick={[Function]}
      type="button"
    >
      3
    </button>
    <button
      className="extra"
      onClick={[Function]}
      type="button"
    >
      +
    </button>
    <button
      className="cero"
      onClick={[Function]}
      type="button"
    >
      0
    </button>
    <button
      className="dot"
      onClick={[Function]}
      type="button"
    >
      .
    </button>
    <button
      className="equal"
      onClick={[Function]}
      type="button"
    >
      =
    </button>
  </div>
</div>
`;
